lang/lang-export.sh

```diff
@@ -16,18 +24,22 @@ if [ -z "$LNG" ]; then LNG=all; fi
 # if 'all' is selected, script will generate all po files and also pot file
 if [ "$LNG" = "all" ]; then
  ./lang-export.sh en
- ./lang-export.sh cz
- ./lang-export.sh de
- ./lang-export.sh es
- ./lang-export.sh fr
- ./lang-export.sh it
- ./lang-export.sh pl
+  for lang in $LANGUAGES; do
+   ./lang-export.sh $lang
+  done
+ #./lang-export.sh cz
```

Comment:
we can probably remove the commented lines

Reply:
Sure we can I had these just as a fall back. on it

Reply:
https://github.com/prusa3d/Prusa-Firmware/pull/3336/commits/6dd7681f6b2e5595c1a265be538dd64ed57c4466